dipy/io/gradients.py

```diff
@@ -1,9 +1,9 @@
-
 from os.path import splitext
 import re
+import tempfile
+from typing import final
```

Comment:
Can you remove this import? it seems to not be used. Thanks!

Reply:
yeesh sorry good cash that was some IDE BS